obsidian.el

```diff
@@ -446,6 +445,26 @@ See `markdown-follow-link-at-point' and
 	 (choice (completing-read "Select file: " results)))
     (obsidian-find-file choice)))
 
+(when (eval-when-compile (require 'hydra nil t))
+  (defhydra obsidian-hydra (:hint nil) "
+Obsidian
+_f_ollow at point   insert _w_ikilink          _q_uit
+_j_ump to note      insert _l_ink
+_t_ag find          _c_apture new note
+_s_earch by expr.   _u_pdate tags/alises etc.
+" ("c" obsidian-capture)
+("f" obsidian-follow-link-at-point)
+("j" obsidian-jump)
+("l" obsidian-insert-link
+ :color blue)
+("q" nil
+ :color blue)
+("s" obsidian-search)
+("t" obsidian-tag-find)
+("u" obsidian-update)
+("w" obsidian-insert-wikilink
+ :color blue)))
```

Comment:
In order to fix using tabs to make the tests pass:
```suggestion
(when (eval-when-compile (require 'hydra nil t))
	(defhydra obsidian-hydra (:hint nil) "
Obsidian
_f_ollow at point   insert _w_ikilink          _q_uit
_j_ump to note      insert _l_ink
_t_ag find          _c_apture new note
_s_earch by expr.   _u_pdate tags/alises etc.
"
		("c" obsidian-capture)
		("f" obsidian-follow-link-at-point)
		("j" obsidian-jump)
		("l" obsidian-insert-link :color blue)
		("q" nil :color blue)
		("s" obsidian-search)
		("t" obsidian-tag-find)
		("u" obsidian-update)
		("w" obsidian-insert-wikilink :color blue)))
```

Reply:
The problem is not tabs:

https://github.com/licht1stein/obsidian.el/runs/7828496340?check_suite_focus=true

Reply:
Only on 27.2

Reply:
I know. It's because you've changed indentation too much. If you make the above changes, then it should work. It's just like a `defun` with a docstring and then the contents, they're also one level deeper.